docs/tutorials/alibabacloud.md

```diff
@@ -194,6 +194,7 @@ spec:
         - --alibaba-cloud-zone-type=public # only look at public hosted zones (valid values are public, private or no value for both)
         - --registry=txt
         - --txt-owner-id=my-identifier
+        - --alibaba-cloud-config-file= # enable sts token 
```

Comment:
Do you need to specify a config file, because after = the value is missing?

Reply:
No.    
Because in the document,  we have already assume ram role to the ecs(ec2) machine.  So the machine have permission to change dns record. 

So the config file is empty is fine.  

@njuettner

Reply:
Got it, thanks